proto/buildbuddy_service.proto

```diff
@@ -168,6 +168,8 @@ service BuildBuddyService {
       returns (github.UnlinkGitHubAccountResponse);
 
   // GitHub app link API
+  rpc GetGitHubAppInstallPath(github.GetGithubAppInstallPathRequest)
```

Comment:
instead of adding another RPC, could the server expose a generic path like /auth/github/install or whatever, then the server calls the appropriate handler? (Having just an HTTP URL would reduce client complexity - we don't have to worry about the loading state of the RPC, and makes it easier to render the URL in multiple places.)

Reply:
I used that approach originally, but it adds another layer of abstraction and complexity (We'd need something like `GitHubAppOauthService` that delegates to the appropriate oauth handler, like how there's GitHubAppService and GitHubApp). There are already many layers of abstraction and complexity, so I thought it was a reasonable trade-off that the client would have some additional complexity. I don't feel strongly though if you'd prefer it's handld in the BE

Reply:
Switching back to a server-side impl is probably more effort than it's worth, but for the future I think we should prefer to have more of the complexity on the server rather than on the client.